reference-architecture/aws-ansible/playbooks/roles/validate-app/library/openshift_facts.py

```diff
@@ -0,0 +1,2162 @@
+#!/usr/bin/python
```

Comment:
I'd try to avoid vendoring openshift_facts if at all possible, I would symlink to the version of openshift-ansible being used rather than vendor.